File: backend/ecommerce/src/main/java/com/bookstore/ecommerce/app/domain/data/Customer.java
package com.bookstore.ecommerce.app.domain.data;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;

@AllArgsConstructor
@Builder
public class Customer {
  @Getter
  private String name;
  @Getter
  private String phone;
  @Getter
  private String deliveryAddress;
}

File: store/app/domain/data/customer.go
package data

type Customer struct {
	Id              string `gorm:"primaryKey"`
	Name            string
	Phone           string
	DeliveryAddress string
}

func (c *Customer) GetId() string {
	return c.Id
}

func (c *Customer) SetId(id string) {
	c.Id = id
}
